src/test-interp.cc

```diff
@@ -16,183 +16,391 @@
 
 #include "gtest/gtest.h"
 
-#include <algorithm>
-#include <functional>
-#include <memory>
-#include <string>
-#include <vector>
-
 #include "src/binary-reader.h"
-#include "src/cast.h"
+#include "src/error-formatter.h"
+
 #include "src/interp/binary-reader-interp.h"
 #include "src/interp/interp.h"
-#include "src/make-unique.h"
 
 using namespace wabt;
+using namespace wabt::interp;
 
-namespace {
-
-interp::Result TrapCallback(const interp::HostFunc* func,
-                            const interp::FuncSignature* sig,
-                            const interp::TypedValues& args,
-                            interp::TypedValues& results) {
-  return interp::ResultType::TrapHostTrapped;
-}
+class InterpTest : public ::testing::Test {
+ public:
+  virtual void SetUp() {}
+  virtual void TearDown() {}
 
-Features s_features;
+  void ReadModule(const std::vector<u8>& data) {
+    Errors errors;
+    ReadBinaryOptions options;
+    Result result =
+        ::ReadModule(data.data(), data.size(), options, &errors, &module_desc_);
```

Comment:
I find the fact that `::` works here it be a little magically.   I'm not quite sure what C++ is doing here.. I guess its find :)

Reply:
Renamed back to `ReadBinaryInterp`, so this isn't necessary anymore.